docs/Using-Mastodon/FAQ.md

```diff
@@ -36,8 +36,15 @@ While Mastodon is compatible with GNU social in terms of server to server commun
 
 Because Mastodon has been created from a blank slate, it is much simpler to have the API mirror internal structures as closely as possible, rather than build an emulation layer. Secondly, the GNU social client API is actually a half-way implementation of the legacy Twitter API - that's the reason why it works with some older Twitter client apps. However, many of those apps are not maintained anymore, the GNU social API does not actually keep up with the real Twitter API and never fully implemented all its features; at the same time, the Twitter API was never meant for a federated service and so obscures some of the functionality.
 
+#### How i can change the default language of the Instance?
```

Comment:
Either remove in favour of #1145 being self-described in the .env file, or refer to it. Modifying `i18n.default_locale` is a bad idea in this application, here `:en` is the most complete localization.